ts/components/ui/Markdown/index.tsx

```diff
@@ -198,6 +205,32 @@ class Markdown extends React.PureComponent<Props, State> {
     }
   }
 
+  public componentWillUnmount(): void {
+    AppState.removeEventListener("change", this.handleAppStateChange);
+  }
+
+  public backgroundState(state: string) {
+    return state.match(/inactive|background/);
+  }
+
+  public handleAppStateChange = (nextAppState: string) => {
+    if (
+      this.backgroundState(this.state.appState) &&
```

Comment:
```suggestion
     this.state.appState !== "active" &&
```

Reply:
@fpersico please check the [latest commit](https://github.com/teamdigitale/io-app/pull/1077/commits/92820dab1c32bbecaef52abe08626f2d63aa1d21).